system/Debug/Exceptions.php

```diff
@@ -177,7 +177,7 @@ public function shutdownHandler()
         ['type' => $type, 'message' => $message, 'file' => $file, 'line' => $line] = $error;
 
         if (in_array($type, [E_ERROR, E_CORE_ERROR, E_COMPILE_ERROR, E_PARSE], true)) {
-            $this->exceptionHandler(new ErrorException($message, $type, 0, $file, $line));
+            $this->exceptionHandler(new ErrorException($message, 0, $type, $file, $line));
```

Comment:
Looks good.

```php
public ErrorException::__construct(
    string $message = "",
    int $code = 0,
    int $severity = E_ERROR,
    ?string $filename = null,
    ?int $line = null,
    ?Throwable $previous = null
)
```

https://www.php.net/manual/en/errorexception.construct.php#refsect1-errorexception.construct-parameters

> severity
    The severity level of the exception.
        Note:
        While the severity can be any int value, it is intended that the [error constants](https://www.php.net/manual/en/errorfunc.constants.php) be used.